1.7/administration/securing-your-cluster.md

```diff
@@ -29,6 +29,44 @@ the other nodes in the cluster via URL routing. For security, the DC/OS cloud
 template allows configuring a whitelist so that only specific IP address
 ranges are permitted to access the admin zone.
 
+#### Steps for Securing Admin Router
+
+By default, Admin Router will permit unencrypted HTTP traffic. This is not
+considered secure, and you must provide a valid SSL certificate and redirect
+all HTTP traffic to HTTPS in order to properly secure access to your cluster.
```

Comment:
I tend to like to use "TLS/SSL" as a qualifier instead of just "SSL." From what I understand, SSL is not really secure anymore and everyone should be using TLS. Though I am not yet sure which we are using or how to configure this. Just a minor comment as I think many people just use SSL when they really mean TLS or SSL.